src/modules/analysis/AnalysisAddComponent.jsx

```diff
@@ -180,9 +224,15 @@ const AnalysisAdd = ({ form, updateForm } : Props) => {
           <FormGroup>
             {LabelFormat('Type analyse', 1)}
             <Col md={2}>
-              <FormControl componentClass="select" placeholder="Velg kategori">
-                <option value="select">Velg kategori</option>
-                <option value="other">...</option>
+              <FormControl
+                componentClass="select"
+                placeholder="Velg kategori"
+                onChange={(e) => updateForm({name: form.analysisTypeId.name, rawValue: e.target.value })}
+              >
+                <option>Velg kategori</option>
+                {store.data.analysisTypes ?
+                  store.data.analysisTypes.map((a) => <option key={a.id} value={a.id}>{a.name}</option>)
+                  : ''}
```

Comment:
Tip:
If you replace the `?` with `&&` (or `!!` if you want the opposite behaviour) then you can skip the `: ''` at the end and reduce the code paths.

```
{store.data.analysisTypes && store.data.analysisTypes.map((a) => <option key={a.id} value={a.id}>{a.name}</option>)}
```